source/docs/software/vision-processing/apriltag/apriltag-intro.rst

```diff
@@ -0,0 +1,233 @@
+.. include:: <isonum.txt>
+
+What Are AprilTags?
+===================
+
+.. image:: images/apriltagrobots_overlay.png
+
+AprilTags are a system of visual tags developed by researchers at the University of Michigan to provide low overhead, high accuracy localization for many different applications.
+
+Application to FRC
+------------------
+
+In the context of FRC, AprilTags are useful for helping your robot know where it is at on the field, so it can align itself to some goal position.
+
+AprilTags have been in development since 2011, and have been refined over the years to increase the robustness and speed of detection.
+
+Starting in 2023, FIRST is providing a number of tags, scattered throughout the field, each at a known :term:`pose`.
+
+All of the tags are from the 36H11 family, which means that:
+
+1) Each tag carries 36 bits of information
+2) It would take at least 11 bit "flips" before one tag could be mistaken for another.
+
+All tags will be printed such that their outer black border is 8.125 inches on a side.
+
+.. image:: images/tag_size.png
+
+For home usage, `these pdf files <https://github.com/rgov/apriltag-pdfs/tree/main/tag36h11>`_ may be printed off and placed around your practice area. Mount them to a rigid backing material to ensure the tag stays flat, as the processing algorithm assumes the tags are flat.
```

Comment:
Please use anonymous links (double underline) for link. Single underline can conflict if you have a duplicate link, and we've standardized on this throughout the documentation.